applications/zkSBT.md

```diff
@@ -0,0 +1,160 @@
+# zkSBT
+- **Team Name:** AsMatch
+- **Payment Address:** 0xfBa1C39f30d6A1D1C35F9551c0f22a0ec6FD30A5 (Ethereum USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+The first framework for SoulBound Tokens (SBTs) was first introduced by Vitalik Buterin, E. Glen Weyl, Puja Ohlhaver, in a whitepaper entitled [“Decentralized Society: Finding Web3’s Soul,”](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4105763) released in May 2022. The whitepaper highlights how Soulbound tokens can be used as a great tool for achieving a fully decentralized society, in view of the amazing possibilities it presents.
+
+SoulBound tokens (SBTs) are non-transferable digital tokens that represent an individual's identity in the Web3 ecosystem. SoulBound tokens (SBTs) can be said to be a form of Digital Curriculum Vitae (CV), as it gives detailed credentials of the user who owns it. These records are stored or distributed via wallets referred to as "Souls". The soul can be said to be a Blockchain wallet that stores publicly accessible, non-transferable tokens.
+
+zkSBT (Zero-Knowledge Soulbond Token) is an enhanced Soulbond Token that uses ZKPs to provide security and privacy features compared to traditional SBTs. Similar to SBTs, zkSBTs are easy to mint and use for verification purposes, offering the same user experience but with added privacy. The users can easily prove their specific identities through zkSBT without inputting any private key to sign the signatures. 
+
+
+### Project Details
+
+AsMatch is the world’s first astrology-based Web3 social-Fi matching app that combines AI-generated content (AIGC) and zero-knowledge proofs (ZKPs) for mobile phones. The zkSBT is one of the most important features of AsMatch. We are supporting more and more SBTs from different blockchain ecosystems, hoping to make it easier for users to import their own credentials. At the same time, this is also an excellent landing scene for the on-chain data of blockchain.
+
+The details about AsMatch zkSBT:
+
+* AIGC Profiles - Generate unique profiles using AIGC technology
+* On-chain Credentials - Privately authorize personal information and token holder verification
+* ZK Verification - Privately verify ownership of assets without revealing the associated wallet address & Embark on a Web3 journey without the need to connect wallets
+
+The following is the workflow designed for our zkSBT service for Polkadot.
+
+For AsMatch Team:
+
+* Deploy ink SBT smart contract to Polkadot blockchain
```

Comment:
Polkadot is just a relay chain and therefore doesn't support smart contract deployment. Did you mean to say deploy to Manta here?